### DISCUSSION TITLE
Refresh local symbolic links (cache)

### DISCUSSION THREAD

**Discussion by maltempi** [treeverse/dvc#10176] [Help] [answered]
Hi folks,

I had this `.dvc/config`

```
[cache]
    dir = /mnt/volume_a/dvc_cache
    shared = group
    type = symlink
```

This makes my dvc-versioned files be a simple symlink, for example:
```sh
$ ls -lah ds/mydataset.csv
[...] ds/mydataset.csv-> /mnt/volume_a/dvc_cache/files/md5/d4/1d8cd98f00b204e9800998ecf8427e
```

Now I want to set my `cache.dir` to `/mnt/volume_b/new_dvc_cache`.

I tried `dvc gc`, `dvc checkout`, `dvc pull`. Nothing updated in the cache. The only way to get my dataset updated was by deleting the `ds/mydataset.csv` and `dvc pull`.

My question is: Is that possible to get it updated without deleting my local file? 

Thank you very much!

**Comment by shcheklein** [comment, marked as answer]
Hey @maltempi , have you tried to use [`dvc checkout --relink`](https://dvc.org/doc/command-reference/checkout#--relink) by chance?

**Reply by maltempi** [reply]
Hello @shcheklein, I haven't noticed the '--relink' before, it works, thank you!

Unfortunately, I haven't had a chance to test it in my actual project, because I have deleted the local symlinks and pulled it again. However, I created a small scenario to try it out and it worked well. If anyone is interested, it is available here: https://github.com/inf-0077/dvc-cache-example/tree/main.

Thank you very much!
